tests/share/transformers/test_oai_dc.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+
+from share.models import SourceConfig
+from share.transformers.oai import OAITransformer
+
+
+class TestSetsFilter:
+
+    TRANSFORMER_CLASS = OAITransformer
+
+    @pytest.fixture
+    def datum(self):
+        return '''
+<record xmlns="http://www.openarchives.org/OAI/2.0/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <header>
+    <identifier>oai:share.osf.io:461B6-A5C-956</identifier>
+    <datestamp>2017-09-12T13:35:17Z</datestamp>
+    <setSpec>au.uow</setSpec>
+  </header>
+  <metadata>
+    <oai_dc:dc xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:oai_dc="http://www.openarchives.org/OAI/2.0/oai_dc/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.openarchives.org/OAI/2.0/oai_dc/ http://www.openarchives.org/OAI/2.0/oai_dc.xsd">
```

Comment:
is this real names/titles of authors/papers?.. would this test appear in search engines. In this case would it be better to use fake name/title

Reply:
Yeah, I just grabbed actual data from staging. You're right, it would be better to use fake data, I'll change it.